pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicpocogkehubfeaturemembership/_generated_object_basicpocogkehubfeaturemembership.golden.yaml

```diff
@@ -15,7 +15,7 @@ spec:
   featureRef:
     name: gkehubfeature-basic-poco-${uniqueId}
   location: global
-  membershipLocation: us-central1
+  membershipLocation: us-west1
```

Comment:
so the issue here was we were using the wrong region in the post submit setup?

Reply:
yes

Reply:
We should probably add this to the mock too in order to mimic the behavior here but that can be done later! Let's just not forget about it? We can put it in an issue.